test/db/formats/pdb

```diff
@@ -146,97 +130,6 @@ EXPECT=<<EOF
 EOF
 RUN
 
-NAME=member1 value of enum R2_TEST_ENUM
```

Comment:
Because they are printed differently now, values as a decimal instead of hex now, the printing of the enum changed into C definition style, so there is no name duplicity etc.